ktlint-core/src/main/kotlin/com/pinterest/ktlint/core/KtLint.kt

```diff
@@ -340,6 +342,15 @@ public object KtLint {
         )
     }
 
+    /**
+     * Get the list of files which will be accessed by KtLint when linting or formatting the given file or directory.
+     */
+    public fun getInputPaths(path: Path): List<Path> =
```

Comment:
For me it works to rename `getInputPaths` to `getEditorConfigFilePaths` and leave out the source file itself which I added solely because the `getInputPath` sounded more generic ;-)
I see no need to the generic methods (`getInputPaths` + `reloadInput`) until we actually have other sources of inputs.